src/core/services/theming/theming.spec.js

```diff
@@ -785,6 +797,40 @@ describe('md-theme directive', function() {
       expect($log.warn.calls.count()).toBe(0);
     });
   });
+
+  it('should accept string as a theme', function() {
+    inject(function($compile, $rootScope, $mdTheming) {
+      var el = $compile('<div md-theme="red"></div>')($rootScope);
+      $rootScope.$apply();
+      $mdTheming(el);
+      expect(el.hasClass('md-default-theme')).toBeFalsy();
+      expect(el.hasClass('md-red-theme')).toBeTruthy();
+    });
+  });
+
+  it('should accept $q promise as a theme', function() {
+    inject(function($compile, $rootScope, $q, $mdTheming) {
+      $rootScope.promise = $q.resolve('red');
```

Comment:
So I thought there was a method on `$mdTheming` that returns a promise (which eventually is the resolve). To make this test more robust and closer to real use, can you use that method instead of just resolve with a string.